mikoba/ViewModels/SSI/SSICredentialViewModel.cs

```diff
@@ -227,7 +227,8 @@ private object FormatCredentialValue(string name, object value)
             {
                 try
                 {
-                    DateTime date = UnixTimestampToDateTime(Convert.ToDouble(value));
+                    var millisecondUnixTimeStamp = Convert.ToDouble(value) * 1000;
+                        DateTime date = UnixTimestampToDateTime(millisecondUnixTimeStamp);
```

Comment:
You could use `var` here and fix the alignment with respect to line 230.

Reply:
done